src/test/static/js-behaviors.js

```diff
@@ -33,4 +36,4 @@ Really.Really.Deep.Behavior = [
 @polymerBehavior
 */
 CustomBehaviorList =
-    [SimpleBehavior, CustomNamedBehavior, Really.Really.Deep.Behavior];
+    [SimpleBehavior, AwesomeBehavior, Really.Really.Deep.Behavior];
```

Comment:
This should still be `CustomNamedBehavior`? Only the jsdoc references `AwesomeBehavior`